cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -102,6 +102,8 @@ limits.add("enforce_acc_limits", bool_t,   0, "If 'True', enforce joint accelera
 limits.add("limits_tolerance",   double_t, 0, "Tolerance around joint position limits where velocities are normalized [degrees]. Must not be zero else DIV/0!", 30, 0.0, 90)
 limits.add("max_vel_lin_base", double_t, 0, "Maximum linear velocity to be commanded to the base in [m/s].", 0.5, 0, 1.5)
 limits.add("max_vel_rot_base", double_t, 0, "Maximum rotational velocity to be commanded to the base in [rad/s].", 0.5, 0, 1.5)
+limits.add("max_lin_twist", double_t, 0, "Maximum input linear velocity, twist command for the inverse kinematics solver in [m/s].", 0.5, 0, 5)
+limits.add("max_rot_twist", double_t, 0, "Maximum input rotational velocity, twist command for the inverse kinematics solver in [rad/s].", 0.5, 0, 6)
```

Comment:
I guess we can reuse the `keep_direction` parameter for the Cartesian/input limiting as well